src/api/controllers/termsAndConditionsController.js

```diff
@@ -0,0 +1,55 @@
+import statusCode from '../../helpers/constants/status.codes';
+import models from '../models';
+
+const { TermsAndConditions } = models;
+
+/**
+ * @class TermsAndConditionController
+ * @description a class to get or update terms and conditions of Authors haven
+ */
+export default class TermsAndConditionController {
+  /**
+   *
+   * @param {Object} req - Request
+   * @param {Object} res  - Response
+   * @param {Function} next -Next
```

Comment:
I don't see next as a parameter on this method

Reply:
Let me remove it .it was a mistake

Reply:
done